artemis-journal/src/main/java/org/apache/activemq/artemis/journal/ActiveMQJournalBundle.java

```diff
@@ -47,4 +47,7 @@
 
    @Message(id = 149004, value = "unable to open file")
    String unableToOpenFile();
+
+   @Message(id = 149005, value = "Record (size={0}) larger than the store maxRecordSize({1})", format = Message.Format.MESSAGE_FORMAT)
```

Comment:
please, use a more user friendly message... something that would make sense to an user...

"Your message is storing too much data as part of properties (maxSize=maxSize). Store large data as part of the MessageBody"

Something like that.. I dunno.. this is just a suggestion.


this is actually the reason I asked you to change the test.. if you ever change the english grammar, translations.. whateverthe test will be broken. Test based on the code.

Reply:
The new message of the exception is: "Message of {0} bytes is bigger than the max record size of {1} bytes. You should try to move large application properties to the message body."